### PR TITLE
feat(table): added draggable variant

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly/issues/4065

### CODE REVIEW

**Review by mcarrano — APPROVED** [review]
Looks great @mcoker

File: src/patternfly/components/Table/table.scss [hunk 1]
```diff
@@ -823,6 +838,13 @@
   }
 }
 
+// Draggable
+.pf-c-table__draggable {
+  .pf-c-button {
+    margin: var(--pf-c-table__draggable--c-button--MarginTop) var(--pf-c-table__draggable--c-button--MarginRight) var(--pf-c-table__draggable--c-button--MarginBottom) var(--pf-c-table__draggable--c-button--MarginLeft);
```
Comment (mattnolting) [inline src/patternfly/components/Table/table.scss:844]:
```suggestion
    margin: var(--pf-c-table__draggable--c-button--MarginTop) var(--pf-c-table__draggable--c-button--MarginRight) var(--pf-c-table__draggable--c-button--MarginBottom) var(--pf-c-table__draggable--c-button--MarginLeft);
    cursor: grab;        
    
    &:active {      
      cursor: grabbing;    
    }  
  }    
```

**Review by mattnolting — COMMENTED** [review]
Looks great, just the grab cursor for consistency.

**Review by mattnolting — APPROVED** [review]
LPTM 🚀

**Pull request merged**